src/model/trust.model.ts

```diff
@@ -79,6 +79,10 @@ export interface TrustIndividual {
   sa_address_postal_code?: string;
   sa_address_care_of?: string;
   sa_address_po_box?: string;
+  still_involved?: string,
```

Comment:
In the corresponding legal entity changes I have named these "stillInvolved" for the page model and "is_corporate_still_involved_in_trust" so as to:

1) clearly distinguish them in the web

2) reflect what is in the other micro services where the name is clearly related to the corresponding field

Which is the best course of action here?

Reply:
i can change it to be of is_....still_involved_in_trust format

Reply:
Having problem passing this value to the sdk so might use "still_involved" as well so that it is distinct at every stage.

Reply:
updated the name